roles/usb_autorunner/files/autorunner.sh.j2

```diff
@@ -1,45 +1,48 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
+TEMP_PATH=/tmp/autorunner/
 AUTORUN_CONTENT_PATH={{ usb_autorunner_install_path }}/workdir
 PRIVATE_CERT={{ usb_autorunner_install_path }}/certificate/autorunner.pem
 PUBLIC_CERT={{ usb_autorunner_install_path }}/certificate/autorunner.pub.pem
 
-AUTORUN_FILENAME=startup.zip.enc
-KEY_FILENAME=secret.key.enc
+AUTORUN_FILENAME=autorun.zip.enc
+SECRET_KEY_FILENAME=secret.key.enc
 
-USB_DATA_PATH=$1
+MOUNT_POINT=$1
 
-if [ ! -r $USB_DATA_PATH/$KEY_FILENAME ]
+if [ ! -r $MOUNT_POINT/$SECRET_KEY_FILENAME ]
 then
-  echo "key file is missing, exiting"
+  echo "Secret key file ‘$SECRET_KEY_FILENAME’ is missing at path $MOUNT_POINT/"
```

Comment:
Looks like this is not exactly single quote but some other character.
Could you make sure those are " ' "